Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -126,6 +126,11 @@ class BrowserViewController: UIViewController {
     override func traitCollectionDidChange(previousTraitCollection: UITraitCollection?) {
```

Comment:
I wonder why we're using this instead of `viewWillTransitionToSize`? If we used `viewWillTransitionToSize` instead, we could even do something like this to "undo" the hack:

```
    override func viewWillTransitionToSize(size: CGSize, withTransitionCoordinator coordinator: UIViewControllerTransitionCoordinator) {
        coordinator.animateAlongsideTransition({ animation in
            if let tab = self.tabManager.selectedTab {
                // scroll offset by 1
            }
        }, completion: { context in
            if let tab = self.tabManager.selectedTab {
                // reset scroll offset
            }
        })
    }
```